### ISSUE TITLE
Adding memory and received time details

### ISSUE THREAD

**Issue by sonec** [mpentler/boinctui-extended#10]
Hi,

I'm not sure if you're accepting any code changes to this project, however I've added code to my local branch to:

1. Show the memory details (swap_size) of a given task in MB/GB, both in the main window and in the task info window. 
2. Show the received time of a given task to the main window.

Let me know if you'd like to take a look at the code changes at all. 

Apologies, I'm new to trying to submit code to a project so not sure if I'm following the correct process.

**Comment by mpentler** [comment]
Hey,

I am very new to this too! I’ll accept submissions and I’ll have a look today.

I will say I don’t really have a test bed anymore as I’ve kinda given up BOINC..


On 9 Jul 2020, at 13:55, sonec <notifications@github.com> wrote:

﻿

Hi,

I'm not sure if you're accepting any code changes to this project, however I've added code to my local branch to:

  1.  Show the memory details (swap_size) of a given task in MB/GB, both in the main window and in the task info window.
  2.  Show the received time of a given task to the main window.

Let me know if you'd like to take a look at the code changes at all.

Apologies, I'm new to trying to submit code to a project so not sure if I'm following the correct process.

**Comment by sonec** [comment]
No worries!

I've just tried pushing a branch to this repository but I'm getting an access denied message.

However, if you're no longer working on this project, would it be more appropriate for me to create a fork?

Let me know what you think.

**Comment by mpentler** [comment]
I mean, best to just submit a PR - I'm happy to take that way and add submissions in to the master branch. I'm very bad with source control ;)

Long-term I would like somebody else who cares more about BOINC these days to take-over this project. I really only got into it because I found myself using it and wished it had everyone's additions in one version. I'm surprised people find it useful really.

**Comment by sonec** [comment]
Hi, I've just submitted the pull request for the code change I've made - #11 

I had to google how to do that, turns out the original method I tried was very incorrect. 

I find it incredibly useful, it allows me to manage any BOINC clients via terminal and SSH, which I find very helpful. Thank you for this project!

When I find some time, I'm tempted to implement all the RPC functionality that's missing, for instance being able to configure the Computing Preferences. However, that will probably take a while.

In the meantime, I'll leave pull request #11 for you to review. If you wish, feel free to close this issue.
